scipy/signal/_fir_filter_design.py

```diff
@@ -7,13 +7,15 @@
 
 import numpy as np
 from numpy.fft import irfft, fft, ifft
+from scipy.signal.windows import get_window
 from scipy.special import sinc
 from scipy.linalg import (toeplitz, hankel, solve, LinAlgError, LinAlgWarning,
                           lstsq)
 from scipy.signal._arraytools import _validate_fs
 
 from . import _sigtools
 
+
 __all__ = ['kaiser_beta', 'kaiser_atten', 'kaiserord',
            'firwin', 'firwin2', 'remez', 'firls', 'minimum_phase']
```

Comment:
Try adding ``'fwind1'`` to ``__all__`` to get rid of the refguide_check warning.

Reply:
I added the suggested change , and stil the CI fails - what could cause this?